examples/video_player/src/lib.rs

```diff
@@ -0,0 +1,228 @@
+use eframe::egui::{self, Grid};
+use eframe::egui::{CentralPanel, TextEdit};
+use humanize_duration::{prelude::*, Truncate};
+use kornia::image::{Image, ImageSize};
+use kornia::imgproc::resize::resize_fast;
+use kornia::io::stream::video::{ImageFormat, SeekFlags, State, VideoReader};
+use std::path::PathBuf;
+
+const SLIDER_HEIGHT: f32 = 20.;
+
+pub struct MyApp {
+    file_path: String,
+    video_reader: Option<VideoReader>,
+    slider_pos: u64,
+    playback_speed: u32,
+}
+
+impl Default for MyApp {
+    fn default() -> Self {
+        Self {
+            file_path: String::new(),
+            video_reader: None,
+            slider_pos: 0,
+            playback_speed: 1,
+        }
+    }
+}
+
+impl eframe::App for MyApp {
+    fn update(&mut self, ctx: &eframe::egui::Context, _frame: &mut eframe::Frame) {
+        ctx.request_repaint();
```

Comment:
not quite sure if the issue is here, but i'm getting the app with a constant blinking

Reply:
worth asking in egui discord